vllm/__init__.py

```diff
@@ -12,10 +12,10 @@
 from vllm.pooling_params import PoolingParams
 from vllm.sampling_params import SamplingParams
 
-# UPSTREAM SYNC: use the current downstream.
-__version__ = "0.5.0"
+from .version import __version__
 
 __all__ = [
+    "__version__",
```

Comment:
where did the version's value go?

Reply:
Its moved to `version.py` + then imported